test/e2e/gameserverallocation_test.go

```diff
@@ -44,6 +44,9 @@ func TestCreateFleetAndGameServerAllocate(t *testing.T) {
 	for _, strategy := range fixtures {
 		strategy := strategy
 		t.Run(string(strategy), func(t *testing.T) {
+			if strategy == apis.Distributed {
```

Comment:
Given it's difficult to stub out the global `apiHooks` - the other option could be than rather than / in addition to skipping these e2e tests, add e2e tests just for Autopilot that would test to see that we get the appropriate error on `apis.Distributed`.

Then the branch above is tested, without having to do the stubbing. WDYT?

Reply:
Yup, agreed, will add.